state/stateConsensus.go

```diff
@@ -87,8 +87,11 @@ func (s *State) Process() (progress bool) {
 		if now-s.StartDelay > s.StartDelayLimit {
 			if s.DBFinished == true {
 				s.RunLeader = true
-				s.StartDelay = now // Reset StartDelay for Ignore Missing
-				s.IgnoreMissing = true
+				if !s.IgnoreDone {
+					s.StartDelay = now // Reset StartDelay for Ignore Missing
```

Comment:
Just to be _completely_ paranoid, if `IgnoreDone` is `true`, then `StartDelay` does *not* get reset, correct?